redis/cluster.py

```diff
@@ -727,12 +738,14 @@ def _determine_nodes(self, *args, **kwargs):
             return [node]
 
     def _should_reinitialized(self):
-        # In order not to reinitialize the cluster, the user can set
-        # reinitialize_steps to 0.
-        if self.reinitialize_steps == 0:
+        # To reinitialize the cluster on every MOVED error,
+        # set moved_reinitialize_steps to 1.
+        # To avoid reinitializing the cluster on moved errors, set
+        # moved_reinitialize_steps to 0.
+        if self.moved_reinitialize_steps == 0:
             return False
         else:
-            return self.reinitialize_counter % self.reinitialize_steps == 0
+            return self.moved_reinitialize_counter % self.moved_reinitialize_steps == 0
```

Comment:
IMHO we shouldn't rename this at this time.There isn't anything to be gained.

Reply:
rollbacked